mmv1/third_party/terraform/tests/resource_compute_backend_service_test.go.erb

```diff
@@ -1181,6 +1181,8 @@ resource "google_compute_backend_service" "foobar" {
   health_checks = [google_compute_http_health_check.zero.self_link]
 
   cdn_policy {
+    negative_caching = false
```

Comment:
I think this belongs to a 1-step test, can we instead modify the test to create with non-empty values first and then update to empty values 2nd?

Reply:
Yes, let me update the tests